### ISSUE TITLE
does not work with virtual camera on mac (OBS 28)

### ISSUE THREAD

**Issue by bkw** [exeldro/obs-downstream-keyer#18]
- OBS 28.0.2 (mac)
- Downstream-keyer release 0.2.5
- OS macOS Monterey 12.6
- Machine: MacBook Pro (13-inch, M1, 2020)

The Preview shows the source, but the virtual cam doesn't. Has also been reported in the forum, duplicating here for visibility.

**Comment by justuseapen** [comment]
@bkw Can you share your installation process for this plugin please?

**Comment by bkw** [comment]
@justuseapen I'm not doing anything exotic, and I also understand from the author's comment that the issue discussed here lies with OBS, not the plugin. But if it helps you in any way, here's what I did (I don't think I did anything special) for the installation:

* Download `downstream-keyer-0.2.5-macos.zip` from https://obsproject.com/forum/resources/downstream-keyer.1254/version/4398/download?file=86051
* Unpack `downstream-keyer-0.2.5-macos.zip`
* Run `downstream-keyer-0.2.5-macos-arm64.pkg` (confirm gatekeeper exception)
* Install into default location
* Restart OBS
* Enable `Downstream Keyer` in Menu `Docks`.

If you have more questions about running the plugin, I think [the forum](https://obsproject.com/forum/threads/downstream-keyer.142247/) is a better place to ask these. Good luck.
